crates/node/service/src/actors/sequencer/actor.rs

```diff
@@ -161,7 +161,17 @@ impl<AB: AttributesBuilder> SequencerActorState<AB> {
         }
 
         let unsafe_head = *unsafe_head_rx.borrow();
-        let l1_origin = self.origin_selector.next_l1_origin(unsafe_head).await?;
+        let l1_origin = match self.origin_selector.next_l1_origin(unsafe_head).await {
+            Ok(l1_origin) => l1_origin,
+            Err(err) => {
+                warn!(
+                    target: "sequencer",
+                    ?err,
+                    "Temporary error occurred while selecting next L1 origin. Re-attempting on next tick."
+                );
+                return Ok(())
+            }
+        };
```

Comment:
Note that we don't really make a difference between what errors returned by the L1 origin selector are temporary vs. which aren't. The sequencer should always assume that transient RPC errors will recover.